testsuites/CBLTester/p2p_tests/conftest.py

```diff
@@ -128,9 +127,8 @@ def params_from_base_suite_setup(request):
             log_info("Downloading TestServer ...")
             # Download TestServer app
             testserver.download()
-
             # Install TestServer app
-            if device_enabled:
+            if run_on_device:
```

Comment:
this still rollback the device support. Could you merge the master to your branch and solve the conflicts?

Reply:
I updated and resolved the conflicts, I changed this condition intentionally

Reply:
I don't understand why you make this change. if you intentionally to rollback to the origin, only want to run on-device or not-on-device for all devices, you will need to clean out the code
"device_enabled_list": device_enabled_list,
and update all p2p jobs back to original states.